src/Components/footer/Footer.scss

```
@@ -0,0 +1,37 @@
+@import '../../Styles/common.scss';
+
+.footer{
```

```suggestion
.footer {
```
셀렉터와 중괄호가 붙어 있네요. 전부 찾아서 한칸 띄워주는 방식으로 바꿔주세요!

---

수정했습니다